config_example.sh

```diff
@@ -133,11 +133,11 @@
 
 # Image for target hosts deployment
 #
-#export IMAGE_NAME="CENTOS_8.2_NODE_IMAGE_K8S_v1.21.0.qcow2"
+#export IMAGE_NAME="CENTOS_8.2_NODE_IMAGE_K8S_${KUBERNETES_VERSION}.qcow2"
```

Comment:
This is an example config,  maybe it is redundant ?

Reply:
replaced it with static value